### ISSUE TITLE
Feature Request: Display citation item content

### ISSUE THREAD

**Issue by clsmt** [SublimeText/LaTeXTools#1328]
### Request
When moving the cursor or mouse over an cited item, e.g. in `\cite{abc98}`, open a small window displaying the title, author etc. of that item.

### Reason
This would be very helpful in revising a paper when one needs to recheck references, and moving them around.

**Comment by r-stein** [comment]
I already thought of a popup for [jump-top-anywhere](https://latextools.readthedocs.io/en/latest/keybindings/#jump-to-anywhere), which would give some contextual information about the jumptarget.

**Comment by r-stein** [comment]
I started with the concept of a hover popup [here](https://github.com/r-stein/LaTeXTools/blob/6f4fb7f52cb7ad49269046c22d42bf8710ed6708/jumpto_anywhere.py#L84-L108).
If you want to use the first version you can download it here: [jumpto_anywhere.py](https://raw.githubusercontent.com/r-stein/LaTeXTools/jumptoanywhere_hover/jumpto_anywhere.py) and replace the file in your LaTeXTools package folder (Preferences > Browse Packages...)

**Comment by clsmt** [comment]
How do I use this feature again? I installed the LatexTools 4.0 alpha, located `jumpto_anywhere.py`, then inserted the codes in `class JumptoAnywhereHelper(object)` and  `class JumptoAnywhereCiteHelper(JumptoAnywhereHelper)`.

But when I move cursor or hover over the citation items, nothing happens?

**Comment by r-stein** [comment]
You need to replace the whole file. There are imports at the top and a hover listener at the bottom.

**Comment by clsmt** [comment]
Thanks! It works now.

It seems the info in the popup is repeated like this:
![image](https://user-images.githubusercontent.com/25810085/40889329-b3065e64-6719-11e8-920c-be9d3f6db3c6.png)

**Comment by r-stein** [comment]
Yes in the first approach it just reuses the information of the [`cite_panel_format`](https://github.com/SublimeText/LaTeXTools/blob/master/LaTeXTools.sublime-settings#L581) setting (because that is easily available), which is by default:
```js
"cite_panel_format": ["{author_short} {year} - {title_short} ({keyword})","{title}"],
```
In the future we may add an additional setting for this popup.

**Comment by clsmt** [comment]
Oh I see.
Thanks for the effort! This is very helpful for me.

**Comment by fliptanedo** [comment]
I just wanted to add that this feature is one of my favorite tweaks to LaTeXTools. It is a big help with my workflow. Thank you @r-stein (and @clsmt for requesting this).

**Comment by fliptanedo** [comment]
Following up: it appears that the hover popup no longer works. Might this be because of a change in the LaTeXTools main branch?

Replacing the [jumpto_anywhere.py](https://github.com/r-stein/LaTeXTools/blob/6f4fb7f52cb7ad49269046c22d42bf8710ed6708/jumpto_anywhere.py) file in /Sublime Text/Packages/LaTeXTools/latextools no longer gives the nice mouseover behavior in Sublime Text. 

Is there a way to fix this? It is very appreciated.
